torkeep/src/intTest/java/org/opencadc/torkeep/TorkeepIntTest.java

```diff
@@ -118,33 +114,77 @@ public TorkeepIntTest() {
     }
 
     @Test
-    public void testCleanPutGetSuccess() throws Exception {
+    public void testPutGetSuccessCAOM() throws Exception {
         // create an observation using subject1
-        Observation observation = new SimpleObservation(TEST_COLLECTION, 
-                URI.create("caom:" + TEST_COLLECTION + "/testCleanPutGetSuccess"), SimpleObservation.EXPOSURE);
+        URI ouri = URI.create("caom:" + TEST_COLLECTION + "/testCleanPutGetSuccessCAOM");
+        Observation observation = new SimpleObservation(TEST_COLLECTION, ouri, SimpleObservation.EXPOSURE);
         Plane p = new Plane(URI.create(observation.getURI().toASCIIString() + "/bar"));
         observation.getPlanes().add(p);
 
         Artifact a = new Artifact(URI.create("cadc:FOO/foo"), DataLinkSemantics.THIS, ReleaseType.DATA);
         p.getArtifacts().add(a);
 
-        Part pa = new Part(0);
-        a.getParts().add(pa);
+        deleteObservation(observation.getURI(), subject1, null, null);
+        
+        final URI accMetaChecksum1 = observation.computeAccMetaChecksum(MessageDigest.getInstance("MD5"));
+        putObservation(observation, subject1, 200, "OK");
+        
+        Observation ret = getObservation(observation.getURI(), subject1, 200, null, EXPECTED_CAOM_VERSION);
+        Assert.assertEquals("wrong URI", observation.getURI(), ret.getURI());
+        Assert.assertEquals("wrong ID", observation.getID(), ret.getID());
+        Assert.assertEquals("wrong checksum", accMetaChecksum1, ret.getAccMetaChecksum());
+
+        observation.telescope = new Telescope("FOOSCOPE");
+        final URI accMetaChecksum2 = observation.computeAccMetaChecksum(MessageDigest.getInstance("MD5"));
+        Assert.assertNotEquals(accMetaChecksum1, accMetaChecksum2);
+        
+        updateObservation(observation, subject1, 200, "OK");
+        ret = getObservation(observation.getURI(), subject1, 200, null, EXPECTED_CAOM_VERSION);
+        Assert.assertEquals("wrong URI", observation.getURI(), ret.getURI());
+        Assert.assertEquals("wrong ID", observation.getID(), ret.getID());
+        Assert.assertEquals("wrong checksum", accMetaChecksum2, ret.getAccMetaChecksum());
+        
+        // cleanup (ok to fail)
+        //deleteObservation(observation.getURI(), subject1, null, null);
```

Comment:
Enable cleanup? (comment applies to entire file)

Reply:
it's currently useful to do cleanup before test and not after (so there is some db content to check things manually)